Android/app/src/go/doh/doh.go

```diff
@@ -69,21 +70,23 @@ type Summary struct {
 }
 
 // A Token is an opaque handle used to match responses to queries.
-type Token interface{}
+type Token = interface{}
 
 // Listener receives Summaries.
 type Listener interface {
 	OnQuery(url string) Token
 	OnResponse(Token, *Summary)
 }
 
-// Transport represents a DNS query transport.  This interface is exported by gobind,
-// so it has to be very simple.
+// Transport represents a DNS query transport.
 type Transport interface {
```

Comment:
Can we rename `Transport` to something more meaningful, like `DOHResolver`?
This gets confused with the HTTP Transport.

Reply:
renamed to `doh.Resolver`. Since it's already in the `doh` package, `doh.DoHResolver` sounds redundant.